chibuezeayogu/React-Math-Skills-Game
# Ft add answers component 160793932

**What does the PR do?**
- Add `Answers Component`

**Description of Task to be completed?**
- write component test
- add answers component

**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Math-Skills-Game.gitt)
-  cd into `React-Math-Skills-Game` and checkout to this branch `ft-add-answers-component-160793932`
- run `npm start`
- run `npm test` and ensure all test are passing

**What are the relevant pivotal tracker stories?**
- [#160793932](https://www.pivotaltracker.com/story/show/160793932)

**Screenshots**
- N/A
